bazelrc/bazel-0.14.0.bazelrc

```diff
@@ -0,0 +1,89 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
```

Comment:
We will be releasing configs for 0.14.0 shortly. You can send your change for all the previous versions for now. Please let us know which new flags will be needed for BES for 0.14.0 so that we can include them in that release

Reply:
ok cool. The new flags I need are:

build:results --bes_results_url="https://source.cloud.google.com/results/invocations/"

build:results-local --bes_results_url="https://source.cloud.google.com/results/invocations/"

Reply:
also setting 

build:results --bes_timeout=60s

build:results-local --bes_timeout=60s

would be good.

Reply:
thanks! I'm happy to review the changes if you want.